third_party/npm_redux-act/README.chromium

```diff
@@ -0,0 +1,4 @@
+Name: redux-act
+URL: https://github.com/pauldijou/redux-act
+License: Apache License 2.0
```

Comment:
I haven't found a good place to document this, but I've been trying to keep these in the standard [SPDX format](https://spdx.org/licenses/) in case we want to programmatically check licenses in the future. That one one would be `Apache-2.0`

Reply:
thanks, updated